vocabularies/UI.xml

```diff
@@ -702,6 +702,40 @@ The trend is
         </Property>
       </ComplexType>
 
+      <Term Name="Grid" Type="UI.GridDefinitionType" Nullable="false" AppliesTo="EntityType">
+        <Annotation Term="UI.ThingPerspective" />
+        <Annotation Term="Core.Description" String="Visualization of a data grid" />
```

Comment:
If it is i a "data grid", why not use that as term name?

Reply:
good idea